onnx/shape_inference/implementation.cc

```diff
@@ -116,6 +116,35 @@ void checkShapesAndTypes(const TypeProto& inferredType, const TypeProto& existin
   }
 }
 
+// TypeProto_Tensor or TypeProto_SparseTensor
```

Comment:
add sequence to the comment

Reply:
Actually only TypeProto_Tensor or TypeProto_SparseTensor and will use generateSymbolicShape? Sequence will merely be expanded in materializeSymbolicShape.

Reply:
you are right